src/router.js

```diff
@@ -130,7 +130,7 @@ const router = createRouter({
         {
           path: 'pools',
           name: 'PacbioPoolIndex',
-          component: PacbioPoolIndex,
+          component: PacbioPoolIndexToggleWithFeatureFlag,
```

Comment:
Instead of a separate component to handle the feature flag could we handle it in here? So the router decides which component to show?
Im not sure if it is possible because its asynchronous but might be cleaner.

Reply:
I had a look into it and we can probably use router navigation guards like 'beforeEach' , but that will called for every routing which I thought won't be appropriate.

Reply:
To make this work, I think we should have a way to query feature flag status. Currently we are managing it through useSWRVv from swrv library., but that seems to work only within a component context it seems.

Reply:
Sounds like we need to look into this. I was never sure about the use of swrv as it is another dependency.